src/main/java/dukepkg/Parser.java

```diff
@@ -217,12 +217,21 @@ public Command constructCommand(String command) throws DukeException {
                 case "event":
                     validateAddTaskCommandLength(arr);
                     return Command.getAddTaskCommand(arr);
+                case "find":
```

Comment:
Good that you are using switch statements instead of multiple`if-else` statements :+1: